### ISSUE TITLE
Implement "Digital wallets" settings screen UI

### ISSUE THREAD

**Issue by frosso** [Automattic/woocommerce-payments#1730]
## Description

Blocked by https://github.com/Automattic/woocommerce-payments/issues/1508
See paJDYF-1x5-p2 for latest updates on the screens UI.
Saving of settings will be done in a different task: https://github.com/Automattic/woocommerce-payments/issues/1728

## Acceptance criteria
- The page shows the UI elements, which can be interacted with.

**Comment by bborman22** [comment]
I was wondering a bit about the preview section. I'm spinning my wheels a bit on that today and didn't want to hold things up too much. First, just wanted to see if there was already a solution in mind for this and if not, do you think it would make sense to explore the preview in a separate issue?

**Comment by frosso** [comment]
@bborman22 I think that's good, so things keep moving moving 👍
